### PR TITLE
Initial/draft fix for #33 Chrome's displaying issue

### PR DESCRIPTION
**Pull request by danimesq**
This is all that I know for make it display like a charm in Google
Chrome and other webkit browsers. You can complement this draft fix.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -303,6 +303,7 @@ body {
 	background-image: url(../img/volume_mute.svg);
 }
 #volume-range {
+	-webkit-appearance:none;
```
Comment (nt1m) [inline css/style.css]: Can you add a space after every ":" character ?

File: css/style.css [hunk 2]
```diff
@@ -324,6 +326,25 @@ body {
...
+}
+#volume-range::-webkit-slider-thumb {
+	-webkit-appearance:none;
+	background: #0095DD;
+	border-radius: 50%;
+	height: 16px;
+	width: 16px;
+	border: none;
+}
+#volume-range::-webkit-slider-progress {
```
Comment (nt1m) [inline css/style.css]: Does this even work ? I thought it wasn't supported by webkit

**Comment by danimesq** [comment]
Yes, its supported, but I don't know how do this completly. I've only made the track white and the thumb rounded+cyan-colored. This is only a draft for a future complete work. 
As I know, the one thing that webkit don't support is stylising the progress.

**Comment by danimesq** [comment]
I've added a space every ":" character in rangebar's CSS. My work is finished in this #34 pull request, unless you need more tune ups here.

**Comment by nt1m** [comment]
@DaniellMesquita That means the `#volume-range::-webkit-slider-progress` part doesn't work.

**Comment by nt1m** [comment]
In my opinion, it's better to just remove that rule, until webkit provides an alternative to ::-moz-range-progress.

**Comment by danimesq** [comment]
I've removed that errouneous CSS rule I've that don't exists yet. Thanks for the tip. I'll open a issue  in webkit project requesting an alternative to `::-moz-range-progress`. My work has finished in this initial/draft #34 pull request, unless you need more tune ups here.

**Comment by DaniellMesquita** [comment]
I've removed that errouneous CSS rule I've that don't exists yet. Thanks for the tip. I'll open a issue  in webkit project requesting an alternative to `::-moz-range-progress`. My work has finished in this initial/draft #34 pull request, unless you need more tune ups here.

**Comment by nt1m** [comment]
Thanks !

**Pull request merged**